packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -258,6 +258,7 @@ class LongPressGestureRecognizer extends PrimaryPointerGestureRecognizer {
     super.kind,
     super.supportedDevices,
     super.debugOwner,
+    super.allowedButtonsFilter = _defaultButtonAcceptBehavior,
```

Comment:
Should we add some documentation to this class indicating that the `allowedButtonsFilter` does not give this recognizer the ability to recognize any buttons beyond the ones it already accepts (kPrimaryButton, kSecondaryButton, kTertiaryButton)? 

I point this out because something like `DragGestureRecognizer` which does not have *secondary *tertiary callbacks can use `allowedButtonsFilter` to accept a drag on any button. But `LongPressGestureRecognizer` cannot because it is setup in a way that it has *secondary and *tertiary callbacks. This is also true for `TapGestureRecognizer` I believe.

Reply:
Here it goes.